packages/core/src/connectivity/scp-cf/jwt.ts

```diff
@@ -115,38 +134,14 @@
   options?: VerifyJwtOptions
 ): Promise<JwtPayload> {
   options = { ...defaultVerifyJwtOptions, ...options };
+  await verifyJwtXssec(token, options).catch(err => {
+    throw new ErrorWithCause('Could not verify JWT.', err);
+  });
 
-  const creds = getXsuaaServiceCredentials(token);
-  const verificationKeyURL = getVerificationKeyURL(token);
-  if (verificationKeyURL) {
-    checkDomainVerificationKeyURL(verificationKeyURL, creds.uaadomain);
-  }
```

Comment:
Does this method not use the json webtoken build verification. How is this different to the xssec validation?